src/resolvers/item/resolver.ts

```diff
@@ -1,39 +1,57 @@
 import { Repository } from 'typeorm';
 import { InjectRepository } from 'typeorm-typedi-extensions';
-import { Resolver, Query, Arg, Int, Mutation } from 'type-graphql';
+import { Resolver, Query, Arg, Int, Mutation, Ctx, FieldResolver, Root, Authorized } from 'type-graphql';
 
 import Item from '@/entities/Item';
+import User from '@/entities/User';
 
+import { Context } from '../types';
 import { AddItemInput, UpdateItemInput } from './types';
 
 @Resolver(Item)
 export class ItemResolver {
   constructor(
     @InjectRepository(Item) private readonly itemsRepository: Repository<Item>,
+    @InjectRepository(User) private readonly userRepository: Repository<User>,
   ) {}
 
-  @Query(() => String)
-  public hello() {
-    return '1233';
-  }
-
   @Query(() => Item, { nullable: true })
   public async item(@Arg('itemId', () => Int) itemId: number) {
     return this.itemsRepository.findOne({ id: itemId });
   }
 
   @Query(() => [Item])
   public async items() {
-    return this.itemsRepository.find();
+    return this.itemsRepository.find({
+      order: {
```

Comment:
Do we need order condition here?

Reply:
no ＸＤ

Reply:
You should support Graphql `Filter` input so the frontend can decide what he wants to query.

Like this but we can find if a better way to implement it.
https://www.howtographql.com/graphql-js/8-filtering-pagination-and-sorting/